### PR TITLE
Sezzle updated icon

### PR DESCRIPTION
**Pull request by rossdl**
Update the icon for recent Sezzle rebrand

### CODE REVIEW

File: app/assets/images/payment_icons/sezzle.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...or="#00B874" offset="0%"/><stop stop-color="#29D3A2" offset="51.26%"/><stop stop-color="#53DFB6" offset="68.17%"/><stop stop-color="#9FF4D9" offset="100%"/></linearGradient><linearGradient x1="-.003%" y1="49.999%" x2="99.999%" y2="49.999%" id="pi-sezzle-linearGradient-4"><stop stop-color="#FCD77E" offset="0%"/><stop stop-color="#FEA500" offset="52.41%"/><stop stop-color="#FF5B00" offset="100%"/></linearGradient></defs><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g fill-rule="nonzero"><rect fill="#FFF" x="0" y="0" width="38" height="23.222" rx="3.167"/><g transform="translate(11 1)"><path d="M1.417 14.044a4.836 4.836 0 006.84 0l.18-.181c.944-.944-.945-5.896 0-6.84l-7.02 7.02z" fill="url(#pi-sezzle-linearGradient-1)"/><path d="M8.62 6.84l-.182.182c-.943.944.944 5.896 0 6.84L15.46 6.84a4.823 4.823 0 00-3.42-1.416A4.817 4.817 0 008.62 6.84" fill="url(#pi-sezzle-linearGradient-2)"/><path d="M1.417 7.204a4.836 4.836 0 000 6.84L8.62 6.84a4.836 4.836 0 000-6.84L1.417 7.204z" fill="url(#pi-sezzle-linearGradient-3)"/><path d="M8.256 14.044a4.836 4.836 0 000 6.84l7.204-7.205a4.836 4.836 0 000-6.84l-7.204 7.205z" fill="url(#pi-sezzle-linearGradient-4)"/></g></g></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/sezzle.svg]: It's missing the border around the icon. An example can be found inside the [contribution document](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

Comment (rossdl) [reply app/assets/images/payment_icons/sezzle.svg]: Thanks @tauthomas01.  Should be fixed now.

File: app/assets/images/payment_icons/sezzle.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...set="100%"/></linearGradient><linearGradient x1="-.003%" y1="49.999%" x2="99.999%" y2="49.999%" id="pi-sezzle-linearGradient-4"><stop stop-color="#FCD77E" offset="0%"/><stop stop-color="#FEA500" offset="52.41%"/><stop stop-color="#FF5B00" offset="100%"/></linearGradient></defs><g transform="translate(1 1)" fill-rule="nonzero" stroke="none" stroke-width="1" fill="none"><rect stroke-opacity=".07" stroke="#000" stroke-width=".96" fill="#FFF" x="-.48" y="-.48" width="36.96" height="22.96" rx="1.92"/><path d="M1.36 13.482a4.643 4.643 0 006.566 0l.174-.174c.906-.906-.907-5.66 0-6.566l-6.74 6.74z" fill="url(#pi-sezzle-linearGradient-1)" transform="translate(10 1)"/><path d="M8.275 6.567l-.174.174c-.906.906.906 5.66 0 6.566l6.74-6.74a4.63 4.63 0 00-3.283-1.36 4.624 4.624 0 00-3.283 1.36" fill="url(#pi-sezzle-linearGradient-2)" transform="translate(10 1)"/><path d="M1.36 6.916a4.643 4.643 0 000 6.566l6.916-6.916a4.643 4.643 0 000-6.566L1.36 6.916z" fill="url(#pi-sezzle-linearGradient-3)" transform="translate(10 1)"/><path d="M7.926 13.482a4.643 4.643 0 000 6.566l6.916-6.916a4.643 4.643 0 000-6.566l-6.916 6.916z" fill="url(#pi-sezzle-linearGradient-4)" transform="translate(10 1)"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/sezzle.svg]: The border radius is slightly different than other icons. Is it possible to adjust it?
[Video demonstration](https://screenshot.click/09-26-u0am9-yw3aq.mp4)

Thank you

Comment (rossdl) [reply app/assets/images/payment_icons/sezzle.svg]: @tauthomas01 hopefully it's fixed now, thank you!

**Comment by tauthomas01** [comment]
Hi @rossdl , I still see a difference in border radius: https://screenshot.click/10-46-qhizz-hapjz.mp4

**Comment by rossdl** [comment]
@tauthomas01 hopefully this is the one

**Comment by rossdl** [comment]
@tauthomas01 thanks for your patience, hopefully _this_ is the one

**Pull request merged**